holochain-overlay/default.nix

```diff
@@ -248,21 +286,35 @@ self: super: {
       windows_x64 = "v4Ntf4zCFs4fYM2rFf0h1rHTxapI/c+A7Qo+H2qPRlE=";
     }
   );
+
   holochain_0-4-0 = super.callPackage ./holochain/default.nix (
     self.selectArchConfig {
-      version = "0.4.0";
+      version = "0.4.0";  # Holochain release version, not holochain --version
```

Comment:
This results in a binary name and version mismatch.
```
$ lair-keystore-0.4.0 --version
lair_keystore 0.5.3
```
We will have to make 2 separate attributes for the release version and the binary suffix.  If the binary suffix is not specified, it defaults to the release version.

Reply:
OK, fixed.